src/pint/toa.py

```diff
@@ -103,6 +103,7 @@ def get_TOAs(
     bipm_version=None,
     include_gps=None,
     planets=None,
+    include_pn=True,
```

Comment:
Is there any need for this? The column can easily be removed with `remove_pulse_numbers`.